### PR TITLE
[DOCS-6839] Add redirects for Content Services 7.1

### PR DESCRIPTION
**Pull request by anxumalo**
This PR relates to:

- https://github.com/Alfresco/docs-alfresco/pull/737
- https://github.com/Alfresco/docs-alfresco/pull/735
- https://github.com/Alfresco/docs-alfresco/pull/750

### CODE REVIEW

File: redirects/7.1/concepts/OOoJodconverter-subsystem-props.html [hunk 1]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/libreoffice/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/OOoJodconverter-subsystem-props.html:1]: I don't see a libreoffice folder

File: redirects/7.1/concepts/auth-ldap-ADtips.html [hunk 2]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/troubleshoot/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/auth-ldap-ADtips.html:1]: troubleshoot isn't a folder, it is a markdown page

File: redirects/7.1/concepts/auth-kerberos-clientconfig.html [hunk 3]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/auth-sync/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/auth-kerberos-clientconfig.html:1]: auth-sync isn't a folder it is a markdown page

Comment (anxumalo) [reply redirects/7.1/concepts/auth-kerberos-clientconfig.html:1]: This is correct - the `.md` files in the code are actually published to what look like folder paths in the live site .. so this page is redirected to https://docs.alfresco.com/content-services/7.1/admin/auth-sync/. HTH

File: redirects/7.1/concepts/auth-ldap-ADtips.html (hunk 2 above)
Comment (anxumalo) [reply redirects/7.1/concepts/auth-ldap-ADtips.html:1]: Correct .. see my later comment: https://docs.alfresco.com/content-services/7.1/admin/troubleshoot/

File: redirects/7.1/concepts/OOoJodconverter-subsystem-props.html (hunk 1 above)
Comment (anxumalo) [reply redirects/7.1/concepts/OOoJodconverter-subsystem-props.html:1]: Good spot @Amy-Hyland - the file has been moved to another product `transform-service/latest/install/`.

I'll verify all the redirects that don't work once this is published to the live site. It's too painful and time-consuming to look at this way. I'll use our link-checker to gather the results and fix across all the versions.

File: redirects/7.1/concepts/JMX-props-config.html [hunk 4]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/security/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/JMX-props-config.html:1]: security is not a folder, it is markdown

File: redirects/7.1/concepts/activemq-config.html [hunk 5]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/activemq/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/activemq-config.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/Share-Doclib-Extend-Intro.html [hunk 6]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/develop/software-architecture/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/Share-Doclib-Extend-Intro.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-custom-example.html [hunk 7]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/develop/share-ext-points/share-config/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-custom-example.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-directorymgt.html [hunk 8]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/auth-sync/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-directorymgt.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-replication-config.html [hunk 9]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/replication/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-replication-config.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-hot-threads.html [hunk 10]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/support-tools/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-hot-threads.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-log-settings.html [hunk 11]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/support-tools/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-log-settings.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-scheduled-jobs.html [hunk 12]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/replication/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-scheduled-jobs.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-system-performance.html [hunk 13]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/support-tools/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-system-performance.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/adminconsole-systemsummary.html [hunk 14]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/admin-console/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/adminconsole-systemsummary.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/admintools-cmm-info.html [hunk 15]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/models/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/admintools-cmm-info.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/admintools-using-cmm.html [hunk 16]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/models/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/admintools-using-cmm.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/amazon-rds.html [hunk 17]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/databases/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/amazon-rds.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/aspect-about.html [hunk 18]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/models/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/aspect-about.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/audit-enable.html [hunk 19]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/repository/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/audit-enable.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/auth-subsystem-defaultauth.html [hunk 20]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/subsystems/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/auth-subsystem-defaultauth.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/email-groupspermissions.html [hunk 21]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/security/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/email-groupspermissions.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/enabling-cors.html [hunk 22]
```diff
@@ -0,0 +1 @@
+/content-services/7.1/config/repository/
```
Comment (Amy-Hyland) [inline redirects/7.1/concepts/enabling-cors.html:1]: not a folder, it is markdown

File: redirects/7.1/concepts/JMX-props-config.html (hunk 4 above)
Comment (anxumalo) [reply redirects/7.1/concepts/JMX-props-config.html:1]: Please see my earlier comments - https://github.com/Alfresco/docs-alfresco/pull/749#discussion_r931313646. This is working as designed. I've resolved all the following comments as there's nothing to change. Thanks @Amy-Hyland

**Review by Amy-Hyland — APPROVED** [review]
Passing review as the link checker will be a faster process than manually verifying links.

**Pull request merged**